lib/ansible/module_utils/ec2.py

```diff
@@ -387,7 +389,11 @@ def ansible_dict_to_boto3_filter_list(filters_dict):
     filters_list = []
     for k, v in filters_dict.items():
         filter_dict = {'Name': k}
-        if isinstance(v, string_types):
+        if isinstance(v, bool):
+            filter_dict['Values'] = [str(v).lower()]
```

Comment:
```suggestion
            filter_dict['Values'] = [str(v).capitalize()]
```
I had to change this to call capitalize() rather than lower() to get it to work for a boolean tag I added using the ec2_ami module:
```
- name: create AMI
  ec2_ami:
    instance_id: "{{ account.value.instance_id }}"
    name: "{{ operating_system }}-{{operating_system_version }}-{{ ansible_date_time.date }}"
    region: "{{ region }}"
    wait: false
    delete_snapshot: true
    state: present
    tags:
      Name: "{{operating_system }}-{{ operating_system_version }}-{{ ansible_date_time.date }}"
      os: "{{ operating_system }}"
      os_version: "{{ operating_system_version }}"
      creation_date: "{{ ansible_date_time.date }}"
      managed_by_ansible: true
      latest: true
```
then I tried searching for it using the facts module:
```
- name: get current latest AMI
      ec2_ami_facts:
        region: "{{ region }}"
        filters:
          "tag:latest": true
```
It didn't find the AMI. But, when I changed it to use capitalize(), resulting in True rather than true, it successfully found the AMI.